src/pages/dashboardTab/components/tabView.js

```diff
@@ -73,6 +73,18 @@ const TabView = ({
   const AddToCartAlertDialogRef = useRef();
 
   const [cartIsFull, setCartIsFull] = React.useState(false);
+  const [columns, setColumns] = React.useState('');
+
+  useEffect(() => {
+    /* hide case_id from viewColumns checkbox */
+    const updateColumns = getColumns(customColumn, classes, data, externalLinkIcon);
+    const index = updateColumns.findIndex((c) => c.label.toLowerCase() === 'case id');
```

Comment:
It's like hard coding only for case_id, can we do it from a bento perspective -

src/bento/dashboardTabData.js and add a new field    
columns
 [
      {
        dataField: 'case_id',
        header: 'Case ID',
        sort: 'asc',
        link: '/case/{case_id}',
        primary: true,
        display: true,
      },
      
      and check for the value of the new field and update it so it can be configurable in the future.